src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
+
+        for (Ball ball: balls) {
+            ball = new Lottery().getRandomBall();
+            System.out.println(ball);
+        }
```

Comment:
The enhanced for-loop does not update the `balls` array with the new `Ball` objects. Instead, it assigns the new `Ball` to the `ball` variable, which is a copy of the array element. To fix this, use a traditional for-loop to iterate over the array indices and assign the new `Ball` objects directly to the array elements.